v2/inc/c_logging/log_internal_error.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef LOG_INTERNAL_ERROR_H
+#define LOG_INTERNAL_ERROR_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+void log_internal_error_report();
```

Comment:
>(); [](http://example.com/codeflow?start=30&length=3)

add void here if you indeed want no variables.

Reply:
Fixed.